docker-entrypoint.sh

```diff
@@ -91,11 +81,7 @@ if [ "$1" = 'postgres' ]; then
 			echo
 		done
 
-		if ! kill -s TERM "$pid" || ! wait "$pid"; then
-			echo >&2 'PostgreSQL init process failed'
-			exit 1
-		fi
-
+		gosu postgres pg_ctl -D ${PGDATA} -m fast -w stop
```

Comment:
IMO if `pg_ctl` has a decent error message when it fails, we don't need to do anything except let `set -e` do its thing.